modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -2094,16 +2095,7 @@ def clip(self, lower, upper, **kwargs):
             new_modin_frame = self._modin_frame.map(lambda df: df.clip(**kwargs))
         return self.__constructor__(new_modin_frame)
 
-    def corr(self, method="pearson", min_periods=1):
-        if method == "pearson":
-            numeric_self = self.drop(
-                columns=[
-                    i for i in self.dtypes.index if not is_numeric_dtype(self.dtypes[i])
-                ]
-            )
-            return numeric_self._nancorr(min_periods=min_periods)
-        else:
-            return super().corr(method=method, min_periods=min_periods)
+    corr = CorrCovBuilder.build_corr_method()
 
     def cov(self, min_periods=None, ddof=1):
         return self._nancorr(min_periods=min_periods, cov=True, ddof=ddof)
```

Comment:
We should probably correct the docstring of _nancorr for `cov` parameter by saying it is always true as of now because corr is implemented in other way.

Reply:
corrected the doc-string